recharges/models.py

```diff
@@ -1,19 +1,18 @@
 import uuid
 
-from django.contrib.postgres.fields import HStoreField
 from django.db import models
 
 
-class DummyModel(models.Model):
+class Recharge(models.Model):
 
     """
     - Nice docstring please
     """
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    product_code = models.CharField(max_length=20)
-    data = HStoreField(null=True, blank=True)
+    amount = models.DecimalField(max_digits=5, decimal_places=2)
```

Comment:
This will allow a max amount of R999.99.  Set max_digits to 9 as discussed with Mike.